Marlin/pins_GEN7_12.h

```diff
@@ -40,9 +67,14 @@
 // Limit Switches
 //
 #define X_STOP_PIN          7
+#define X_MIN_PIN           7   // added 6 NOV 2016
```

Comment:
We have either stop pin or min/max pins. We don't have both. The min or max pin is assigned to the stop pin in `pins.h` based on the homing direction.